bapfs/Content/Conventions/index.md

```diff
@@ -0,0 +1,22 @@
+---
+---
+# Conventions
+
+### Announcing Pandemic Pacificon 2020!
```

Comment:
Since I took so long, want me to take a swing here? Was thinking we update this to a "We had a great pacificon!" instead?